docs/src/content/docs/commands/QWATCH.md

```diff
@@ -87,70 +129,81 @@ This query does the following:
 
-1. Initial state (empty leaderboard):
-   QWATCH response: `[] (empty array)`
+1. Initial state (empty leaderboard): `[]`
 
 2. Player 0 scores 5 points:
 
    ```bash
    127.0.0.1:7379> SET match:100:user:0 5
    ```
 
-   QWATCH response: `[] (no change, score <= 10)`
+   Subscription does not return response as `value < 10`. 
 
 3. Player 1 scores 15 points:
 
    ```bash
    127.0.0.1:7379> SET match:100:user:1 15
    ```
 
-   QWATCH response: `[["match:100:user:1", "15"]]`
+   QWATCH Response:
+   ```bash 
+   qwatch    from SELECT $key, $value WHERE $key like 'match:100:*' and $value > 100 ORDER BY $value asc: `[["match:100:user:1", "15"]]`
+   ```
 
 4. Player 2 scores 20 points:
 
    ```bash
    127.0.0.1:7379> SET match:100:user:2 20
    ```
 
-   QWATCH response: `[["match:100:user:2", "20"], ["match:100:user:1", "15"]]`
+   QWATCH Response:
+   ```bash 
+   qwatch    from SELECT $key, $value WHERE $key like 'match:100:*' and $value > 100 ORDER BY $value asc: `[["match:100:user:2", "20"], ["match:100:user:1", "15"]]`
+   ```
 
 5. Player 3 scores 12 points:
 
    ```bash
    127.0.0.1:7379> SET match:100:user:3 12
    ```
 
-   QWATCH response: `[["match:100:user:2", "20"], ["match:100:user:1", "15"], ["match:100:user:3", "12"]]`
+   QWATCH Response:
+   ```bash 
+   qwatch    from SELECT $key, $value WHERE $key like 'match:100:*' and $value > 100 ORDER BY $value asc: `[["match:100:user:2", "20"], ["match:100:user:1", "15"], ["match:100:user:3", "12"]]`
+   ```
 
 6. Player 4 scores 25 points:
 
    ```bash
    127.0.0.1:7379> SET match:100:user:4 25
    ```
 
-   QWATCH response: `[["match:100:user:4", "25"], ["match:100:user:2", "20"], ["match:100:user:1", "15"]]`
+   QWATCH Response:
+   ```bash 
+   qwatch    from SELECT $key, $value WHERE $key like 'match:100:*' and $value > 100 ORDER BY $value asc: `[["match:100:user:4", "25"], ["match:100:user:2", "20"], ["match:100:user:1", "15"]]`
+   ```
 
 7. Player 0 improves their score to 30:
+
    ```bash
    127.0.0.1:7379> SET match:100:user:0 30
    ```
-   QWATCH response: `[["match:100:user:0", "30"], ["match:100:user:4", "25"], ["match:100:user:2", "20"]]`
-
-This example demonstrates how `QWATCH` provides real-time updates as the leaderboard changes, always keeping clients informed of the top 3 scores above 10, without the need for constant polling.
 
-## Error Handling
+   QWATCH Response:
+   ```bash 
+   qwatch    from SELECT $key, $value WHERE $key like 'match:100:*' and $value > 100 ORDER BY $value asc: `[["match:100:user:0", "30"], ["match:100:user:4", "25"], ["match:100:user:2", "20"]]`
+   ```
 
-- `ERR invalid query`: If the provided query is malformed or unsupported.
-- `ERR syntax error`: If the query syntax is incorrect.
-- `ERR unknown command`: If the `QWATCH` command is not implemented.
-- `ERR max number of subscriptions reached`: If the maximum number of allowed subscriptions is exceeded.
```

Comment:
We shouldn't remove this. Could you please help verify what different kinds of major errors can be returned and add them here?

Reply:
Haven't removed it either. Please check out this link: https://github.com/shashank-priyadarshi/dice/blob/851a9d4cbc57a878bfe484e6f213c4de12e7b872/docs/src/content/docs/commands/QWATCH.md?plain=1#L96